src/inline.css.ts

```diff
@@ -49,44 +45,34 @@ export const inline = style({
         [supportsFlexGap]: {
             margin: 0, // restore
             pointerEvents: 'auto', // restore
-            display: 'inline-flex',
             flexDirection: 'row',
             gap: space,
         },
     },
 });
 
-export const fullWidth = style([
-    inline,
-    {
-        display: ['flex', 'grid'],
-        // style repeated to override the inline @supports style
-        '@supports': {
-            [supportsFlexGap]: {
-                display: ['flex', 'grid'],
-            },
-        },
-    },
-]);
+export const fullWidth = style({
+    /**
+     * @deprecated - this should be "flex"
+     * Once changed, the usages that depend on this behavior should be migrated to use <Grid>
+     * See "inline-cases" private story
+     */
+    display: 'grid',
+});
```

Comment:
I don't understand this change. With the previous code we were using flex in browsers that don't support grid

Reply:
Added a fallback to flex